Sources/ApodiniObserve/ConfiguredLogger.swift

```diff
@@ -0,0 +1,232 @@
+            builtLogger?[metadataKey: "endpoint"] = .dictionary(self.endpointMetadata)
+            
+            // Write exporter metadata
+            builtLogger?[metadataKey: "exporter"] = .dictionary(self.exporterMetadata)
+            
+            /// Prio 1: User specifies a `Logger.LogLevel` in the property wrapper for a specific `Handler`
+            if let logLevel = self.logLevel {
+                builtLogger?.logLevel = logLevel
+                
+                // If logging level is configured gloally
+                if let globalConfiguredLogLevel = storage.get(LoggerConfiguration.LoggingStorageKey.self)?.configuration.logLevel {
+                    if logLevel < globalConfiguredLogLevel {
+                        logger.warning("The global configured logging level is \(globalConfiguredLogLevel.rawValue) but Handler \(self.blackboardMetadata.endpointName) has logging level \(logLevel.rawValue) which is lower than the configured global logging level")
+                    }
+                // If logging level is automatically set to a default value
+                } else {
+                    var globalLogLevel: Logger.Level
+                    #if DEBUG
+                    globalLogLevel = .debug
+                    #else
+                    globalLogLevel = .info
+                    #endif
+                    
+                    if logLevel < globalLogLevel {
+                        logger.warning("The global default logging level is \(globalLogLevel.rawValue) but Handler \(self.blackboardMetadata.endpointName) has logging level \(logLevel.rawValue) which is lower than the global default logging level")
+                    }
+                }
+            }
+            /// Prio 2: User specifies a `Logger.LogLevel`either via a CLI argument or via a `LoggerConfiguration` in the configuration of the `WebService`
+            else if let loggingConfiguraiton = storage.get(LoggerConfiguration.LoggingStorageKey.self)?.configuration {
+                builtLogger?.logLevel = loggingConfiguraiton.logLevel
+            }
+            /// Prio 3: No `Logger.LogLevel` specified by user, use defaul value according to environment (debug mode or release mode)
+            else {
+                #if DEBUG
+                builtLogger?.logLevel = .debug
+                #else
+                builtLogger?.logLevel = .info
+                #endif
+            }
+        } else {
+            // Not pretty, but otherwise ApodiniObserve would need to depend on ApodiniWebsocket
+            if String(describing: exporterTypeMetadata.exporterType) == "WebSocketInterfaceExporter" {
+                // Write connection state
+                builtLogger?[metadataKey: "connectionState"] = .string(connection.state.rawValue)
+                
+                // Write request metadata
+                builtLogger?[metadataKey: "request"] = .dictionary(self.getRequestMetadata(from: connection.request))
```

Comment:
Just noted while looking over it.
Is it correct that in successive updates to the `request` metadataKey, the information is left out?
Above it is initialized like the following:
```swift
builtLogger?[metadataKey: "request"] = .dictionary(self.getRequestMetadata(from: request)
                                                                 .merging(self.getRawRequestMetadata(from: connection.information)) { _, new in new })
```

Reply:
Yep you're right, i somehow missed that while CMD+C and CMD+V :D
We need to reevaluate the RawRequestMetadata as well since eg. the raw HTTPbody could change